tendrl/node_agent/tests/test_alerts_utils.py

```diff
@@ -0,0 +1,23 @@
+from tendrl.node_agent.manager.alert_socket import AlertUtils
+import json
+import pytest
+
+
+class Test_alerts_utils(object):
+    def test_alertsSuccessfulValidate(self):
+        expected_alert = {
+            'Host': '127.0.0.1',
+            'Plugin': 'cpu',
```

Comment:
Yes 'Plugin' is expected to be present in any collectd dispatched threshold alert. But yes, as you are saying I'll have a generic structure for this because gluster and ceph integrations cannot be expected to send alerts in the collectd format I have a schema proposal in https://github.com/Tendrl/specifications/pull/18 could you please review if the structure there is good enough